### ISSUE TITLE
Failed build

### ISSUE THREAD

**Issue by FraH90** [acid1103/PDFInverter#7]
After the pdfbox update, I'm trying to build PDFInverter so it doesn't give anymore the previous error.

However building on windows with "gradlew.bat build" I get this error:

```
> Task :compileJava FAILED
C:\Users\FraH\Documents\GitHub\PDFInverter\src\main\java\org\apache\pdfbox\contentstream\PDFColorInverter.java:180: error: operators has private access in PDFStreamEngine
                OperatorProcessor processor = operators.get(name);
                                              ^
1 error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.4.1/userguide/command_line_interface.html#sec:command_line_warnings
```

What's happening? I'm using a different JDK version that the one you've used for this project?

**Comment by acid1103** [comment]
Hi, FraH. I just published a new release that fixed your previous issue. You can find it [here](https://github.com/acid1103/PDFInverter/releases/tag/0.2.0). If you'd still like to be able to build it yourself, you'll first need to cd into the pdfbox directory inside this project and run the command `mvn clean install`, which requires the maven build tool (`mvn`) to be on your class path. It's a build tool similar to gradle. Once you do that, you can then cd back into the main PDFInverter folder and run `gradlew.bat build`. it should build after that.

So basically, you'll want to run:
```
cd pdfbox
mvn clean install
cd ..
gradlew.bat build
```

**Comment by FraH90** [comment]
> Hi, FraH. I just published a new release that fixed your previous issue. You can find it [here](https://github.com/acid1103/PDFInverter/releases/tag/0.2.0). If you'd still like to be able to build it yourself, you'll first need to cd into the pdfbox directory inside this project and run the command `mvn clean install`, which requires the maven build tool (`mvn`) to be on your class path. It's a build tool similar to gradle. Once you do that, you can then cd back into the main PDFInverter folder and run `gradlew.bat build`. it should build after that.
> 
> So basically, you'll want to run:
> ```
> cd pdfbox
> mvn clean install
> cd ..
> gradlew.bat build
> ```

Perfect, thank you very much Steven!! 
We can close this issue then :)
